### ISSUE TITLE
Separate NuGet packages for different UI frameworks?

### ISSUE THREAD

**Issue by ekdahl** [ClemensFischer/XAML-Map-Control#117]
If I have a WPF application targeting e.g. net6.0-windows10.0.18362.0, the WinUI library is referenced instead of the WPF library (that should be referenced). How can this be solved?
Now, I need to remove the NuGet package and directly reference the DLL instead.

A few thoughts:
- Separate NuGet packages depending of what framework is used (WPF/WinUI/UWP)
- Can UseWPF/UseWinUI element in project file be used?

BR,
Fredrik

**Comment by ClemensFischer** [comment]
How is a WPF app targeting `net6.0-windows10.0.18362.0`? There should be

        <TargetFramework>net6.0-windows</TargetFramework>

in the project file. I can't reproduce the issue.

**Comment by ekdahl** [comment]
Because this is not the only NuGet package used in my application.
My application also uses [this package](https://www.nuget.org/packages/iNKORE.UI.WPF.Modern/0.9.29#dependencies-body-tab), that needs net6.0-windows10.0.18362.0 (probably related to CsWinRT).

Source of project file here:
https://github.com/iNKORE-NET/UI.WPF.Modern/blob/main/source/iNKORE.UI.WPF.Modern/iNKORE.UI.WPF.Modern.csproj

**Comment by ClemensFischer** [comment]
I have created another NuGet package that contains only the WPF versions.

**Comment by ekdahl** [comment]
Thanks a lot, very appreciated!

**Comment by OWiger** [comment]
I've got the same issue with MapProjections.

**Comment by OWiger** [comment]
> Or better, I will drop the original packages and keep only the one(s) with the `.WPF` suffix

This fixed my issue, thanks! :)

**Comment by tipa** [comment]
I am one of the few that are still using UWP :)
Is there any NuGet package targeting UWP? I only see the ones for WPF, WinUI3 & Avalonia on NuGet, but I see all the UWP code is in the repo

**Comment by ClemensFischer** [comment]
No, there is no such package. In contrast to the other platforms, there is apparently no support in Visual Studio projects targeting UWP to automatically generate NuGet packages during build. This is however essential for my release procedure.

**Comment by tipa** [comment]
Ok, thanks for the clarification. Will then reference the project directly until I migrate my app to WinUI3
